docker/build/Dockerfile

```diff
@@ -1,30 +1,46 @@
 FROM ubuntu:20.04
+ARG USER_ID=1000
+ARG GROUP_ID=1000
 
 # install the required dependencies to compile AzerothCore
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt update && apt install -y git cmake make gcc g++ clang libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev libace-6.4.5 libace-dev
+RUN apt-get update && apt-get install -y git cmake make gcc g++ clang mysql-client libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev libace-6.4.5 libace-dev
```

Comment:
why switching to apt-get? apt is the replacement for apt-get from what I remember

Reply:
apt is used only with interactive shells